woocommerce/woocommerce
# Hide state selector from Ethiopia and Rwanda addresses

### All Submissions:


### Changes proposed in this Pull Request:

Hide the state selector for addresses in Ethiopia and Rwanda.

This pull request replaces https://github.com/woocommerce/woocommerce/pull/28796 and https://github.com/woocommerce/woocommerce/pull/28795 (see https://github.com/woocommerce/woocommerce/pull/28795#issuecomment-1296987919).




### How to test the changes in this Pull Request:

Try to make a purchase with a shipping address in Ethiopia, verify that you don't see a state selector and that you can place the order. Repeat with Rwanda.

### Other information:




### FOR PR REVIEWER ONLY:
